script/foreman-debug

```diff
@@ -22,10 +22,11 @@ OPTIONS:
   -d DIR  Directory to place the tarball in (default /tmp/foreman-XYZ)
   -g      Skip generic info (CPU, memory, firewall etc.)
   -a      Do not generate a tarball from the resulting directory
-  -m NUM  Maximum lines to keep for each file (default 5000), zero means no limit
+  -m NUM  Maximum lines to keep for each file, deprecated, use -s
   -j PRG  Filter with provided program when creating a tarball
   -p      Additionally print password patterns being filtered out
   -q      Quiet mode
+  -s SIZE Maximum log set size (current and rotated files, defaults to 10000 kB)
```

Comment:
Oh please reword this to be sure user will get the unit is actually kB:

    -s SIZE Maximum log set size in kB (current and rotated files, defaults to 10 MB)